### DISCUSSION TITLE
Goblin's Diamond

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1245] [Art]
![Schermata 2022-07-17 alle 14 57 39](https://user-images.githubusercontent.com/58088988/179399444-f9b66e61-ac2d-4ee2-81b2-a147d385d995.png)

RUUUUUUNNNNN

More unique than rare ❤️ 

![Schermata 2022-07-17 alle 14 46 20](https://user-images.githubusercontent.com/58088988/179398977-68260d0f-93f0-4158-b5dc-1bacb45d60c0.png)

Where to find?

Or regular NFT or maybe here (?)

![Schermata 2022-07-17 alle 14 49 20](https://user-images.githubusercontent.com/58088988/179399129-1fcd8390-8393-4018-b852-228cd8e9f1c3.png)

![Schermata 2022-07-17 alle 14 49 41](https://user-images.githubusercontent.com/58088988/179399130-5eb0ac35-caf7-408e-a8eb-2cbe91d87dba.png)

Artwork

![goblin_diamond2](https://user-images.githubusercontent.com/58088988/179399016-ad25f25f-fb18-413d-acdd-c11c2c422269.png)

**Comment by adamhannigan** [comment]
Nice suggestion @Q4FOIL 

I would suggest making a more pixelated version so it remains consistent with other art in the game.

For small items, we try keep them 12x12px.

**Comment by Q4FOIL** [comment]
@adamhannigan 

I try to remake diamond

This is the miner

![image](https://user-images.githubusercontent.com/58088988/179701168-7deebad7-0074-4c29-b404-65e7e7972cce.png)

![image](https://user-images.githubusercontent.com/58088988/179701278-c92cca84-9663-49ba-b395-b2e36ce1531d.png)

![Schermata 2022-07-19 alle 10 20 54](https://user-images.githubusercontent.com/58088988/179702847-4cdca042-74de-45c5-8819-d9365abe3e32.png)

**Comment by Q4FOIL** [comment]
20x20px

![image](https://user-images.githubusercontent.com/58088988/179718611-598cebad-60ab-4d17-b8e3-578a73f3529d.png)

100x100px

![image](https://user-images.githubusercontent.com/58088988/179718807-1094946c-8d5b-49fd-b8f2-7ca16223ab5a.png)

![image](https://user-images.githubusercontent.com/58088988/179719376-5af11bef-786d-4da6-b7ca-da9b08438605.png)


@adamhannigan
